lib/rubocop/cop/rspec/single_argument_message_chain.rb

```diff
@@ -0,0 +1,53 @@
+module RuboCop
+  module Cop
+    module RSpec
+      # Checks that chains of messages contain more than one element.
+      #
+      # @example
+      #   # bad
+      #   allow(foo).to receive_message_chain(:bar).and_return(42)
+      #
+      #   # good
+      #   allow(foo).to receive(:bar).and_return(42)
+      #
+      #   # also good
+      #   allow(foo).to receive(:bar, :baz)
+      #   allow(foo).to receive("bar.baz")
+      #
+      class SingleArgumentMessageChain < Cop
+        MESSAGE = 'Use `%<recommended_method>s` instead of calling ' \
+          '`%<called_method>s` with a single argument'.freeze
+
+        MESSAGE_CHAIN_METHODS = [
+          :receive_message_chain,
+          :stub_chain
+        ].freeze
```

Comment:
This is in `MessageChain` too. Think we can add a new chain entry to the `RSpec::Language` definitions file?